text/0118-search-api.md

```diff
@@ -1152,16 +1155,27 @@ Added to the search response when `facets` is set for a search query. It contain
 
 If a field distributed as a facet contains no value, it is returned as a `facetDistribution` field with an empty object as value.
 
-> See [3.1.4. `facet`](#314-facets) section.
+> See [3.1.4. `facets`](#314-facets) section.
+
+#### 3.2.10. `facetStats`
+
+- Type: Object
+- Required: False
+
+When using the `facets` parameter, the distributed facets that stricly contain numeric values are displayed in a `facetStats` object that contains, per facet, the numeric `min` and `max` values of the hits returned by the search query.
```

Comment:
We don't ignore a facet as soon as one of the hit contains a non-numeric value, but if all of the hits contain non-numeric values.

```suggestion
When using the `facets` parameter, the distributed facets that contain some numeric values are displayed in a `facetStats` object that contains, per facet, the numeric `min` and `max` values of the hits returned by the search query.

If none of the hits returned by the search query have a numeric value for a facet, this facet is not part of the `facetStats` object.
```

Reply:
Thanks, @dureuill!